package.json

```diff
@@ -4,9 +4,9 @@
   "dependencies": {
     "@edx/edx-bootstrap": "1.0.4",
     "@edx/edx-proctoring": "^1.5.0",
-    "@edx/frontend-auth": "^6.0.2",
+    "@edx/frontend-auth": "9.0.2",
```

Comment:
Are we intentionally removing the `^` in front of 9.0.2?

Reply:
Ah no, I'll fix that